ote/src/cljs/ote/ui/table.cljs

```diff
@@ -77,7 +77,7 @@
              (doall
                (for [{:keys [name width col-style format read]} headers
                      :let [value ((or format identity) (if read (read row) (get row name)))]]
-                 ^{:key (str name)}
+                 ^{:key (str name "-" i)}
```

Comment:
If the names are not unique, adding index won't fix it in all cases. https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318